clients/githubrepo/client.go

```diff
@@ -44,7 +45,7 @@ func (client *Client) InitRepo(owner, repoName string) error {
 	}
 	client.repo = repo
 
-	// Init tarballHandler.
+	// Init tarballHandler
```

Comment:
nit: i thought we keep periods at end, some mismatches in different messages.

Reply:
My bad. We do keep periods at the end.